x-pack/plugins/ml/public/alerting/job_selector.tsx

```diff
@@ -91,11 +108,24 @@ export const JobSelectorControl: FC<JobSelectorControlProps> = ({
               },
             ]
           : []),
+
         {
           label: i18n.translate('xpack.ml.jobSelector.jobOptionsLabel', {
             defaultMessage: 'Jobs',
           }),
-          options: jobIdOptions.map((v) => ({ label: v })),
+          options: [
+            ...(createJobUrl
```

Comment:
Can you add in a check for the `canCreateJob` capability here?

Reply:
Added in https://github.com/elastic/kibana/pull/174016/commits/92ad113125c06479e300eb8b3cd60ffd385dfe5f